src/modules/sih/aero.hpp

```diff
@@ -0,0 +1,407 @@
+ * This class can model
+ * - full 360 deg angle of attack lift, drag, and pitching moment.
+ * - wings with aspect ratio from 0.1666 up to 6, (gliders would perform poorly for instance).
+ * - control surface (flap) deflection up to 70 degrees.
+ * - unlimited flap chord, (the elevator could take the entire tailplane).
+ * - stall angle function of the flap deflection.
+ * - effect of the flap deflection on lift, drag, and pitching moment.
+ * - any dihedral angle, the fin is modeled as a wing with dihedral angle of 90 deg.
+ * - slipstream velocity (velocity from the propeller) using momentum theory.
+ */
+
+
+
+#pragma once
+
+#include <matrix/matrix/math.hpp>   	// matrix, vectors, dcm, quaterions
+#include <conversion/rotation.h>    	// math::radians,
+// #include <lib/mathlib/mathlib.h>
+#include <math.h>
+
+// class Aerodynamic Segment ------------------------------------------------------------------------
+class AeroSeg
+{
+private:
+	// Table 3.1 SEMI-EMPIRICAL COEFFICIENTS FOR RECTANGULAR FLAT PLATES
+	static constexpr const int N_TAB = 12;
+	// static constexpr const float AR_tab[N_TAB] = {0.1666f,0.333f,0.4f,0.5f,1.0f,1.25f, 2.0f, 3.0f, 4.0f, 6.0f};
+	// static constexpr const float ale_tab[N_TAB] = {3.00f,3.64f,4.48f,7.18f,10.20f,13.38f,14.84f,14.49f,9.95f,12.93f,15.00f,15.00f};
+	// static constexpr const float ate_tab[N_TAB] = {5.90f,15.51f,32.57f,39.44f,48.22f,59.29f,21.55f,7.74f,7.05f,5.26f,6.50f,6.50f};
+	// static constexpr const float afle_tab[N_TAB] = {59.00f,58.60f,58.20f,50.00f,41.53f,26.70f,23.44f,21.00f,18.63f,14.28f,11.60f,10.00f};
+	// static constexpr const float afte_tab[N_TAB] = {59.00f,58.60f,58.20f,51.85f,41.46f,28.09f,39.40f,35.86f,26.76f,19.76f,16.43f,14.00f};
+	// static constexpr const float afs_tab[N_TAB] = {49.00f,54.00f,56.00f,48.00f,40.00f,29.00f,27.00f,25.00f,24.00f,22.00f,22.00f,20.00f};
+
+	// deflection effectiveness curve fitted as second order
+	static constexpr const float eta_poly[] = {0.0535f, -0.2688f, 0.5817f}; 	// 1/rad
+
+	// aerodynamic and physical constants
+	static constexpr const float P0 = 101325.0f;	// pressure at sea level [N/m^2]=[Pa]
+	static constexpr const float R = 287.04f;		// real gas constant for air [J/kg/K]
+	static constexpr const float T0_K = 288.15f;	// temperature at sea level [K]
+	static constexpr const float TEMP_GRADIENT  = -6.5e-3f;    // temperature gradient in degrees per metre
+
+	static constexpr const float KV = M_PI_F; 	// total vortex lift parameter
+	static constexpr float CD0 = 0.04f; 		// no lift drag coefficient
+	static constexpr float CD90 = 1.98f; 		// 90 deg angle of attack drag coefficient
+	static constexpr float AF_DOT_MAX = M_PI_F / 2.0f;
+	static constexpr float ALPHA_BLEND = M_PI_F / 18.0f; 	// 10 degrees
+
+	// here we make the distinction of the plate (i.e. wing, or tailplane, or fin) and the segment
+	// the segment can be a portion of the wing, but the aspect ratio (AR) of the wing needs to be used
+	float alpha; 		// angle of attack [rad]
```

Comment:
```suggestion
	float _alpha; 		// angle of attack [rad]
```
For the rest as well